Core/Core/Submissions/APISubmission.swift

```diff
@@ -214,34 +212,6 @@ extension APISubmissionCommentAuthor {
     }
 }
 
-extension APISubmissionUser {
```

Comment:
We are trusting that an APIUser never breaks APISubmissionUser which will be hard to validate but 🤞

Reply:
they use the same json serialization function and we needed another property that made SubUser a larger subset of User, so 🤞.